gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceSource.java

```diff
@@ -32,6 +32,7 @@
 import java.util.Set;
 
 import java.util.stream.Collectors;
+import lombok.SneakyThrows;
```

Comment:
Unused import?

Reply:
Thanks! It was unused import. 
We are using it now.